seed/challenges/01-front-end-development-certification/basic-ziplines.json

```diff
@@ -202,9 +202,9 @@
       "challengeType": 3,
       "titleRu": "Создайте сайт-портфолио",
       "descriptionRu": [
-        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a> которое функционально соответствует вот этому: <a href='http://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>http://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
+        "<strong>Задание:</strong> Создайте приложение <a href='https://codepen.io' target='_blank'>CodePen.io</a> которое функционально соответствует вот этому: <a href='https://codepen.io/FreeCodeCamp/full/VemmoX/' target='_blank'>https://codepen.io/FreeCodeCamp/full/VemmoX/</a>.",
```

Comment:
Shouldn't the link here be changed to https://codepen.io/hallaathrad/full/vNEPpL? I used Google Translate on the surrounding text and it appears this is the personal portfolio page.